tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -44,6 +44,19 @@ def run(ceph_cluster, **kwargs) -> int:
     verify_image = bool(config.get("verify_cluster_health", False))
     orch = Orch(cluster=ceph_cluster, **config)
 
+    # Clean up the ceph-qe-scripts to prepare for the execution of RGW singlesite and multisite test suites.
+    rgw_tests = "/home/cephuser/rgw-tests"
+    for node in ceph_cluster.get_nodes():
+        out, err = node.exec_command(cmd=f"ls -l {rgw_tests}", check_ec=False)
+        if out:
+            node.exec_command(sudo=True, cmd=f"rm -rf {rgw_tests}/*", check_ec=True)
```

Comment:
Hi @obannak ,
could you please remove the directory as well please.. along with the subdirectories.. our code will skip if rgw-tests or rgw-ms-tests dir is present..
https://github.com/red-hat-storage/cephci/blob/main/tests/rgw/sanity_rgw_multisite.py#L395

may be you can remove the * at the end like below..
cmd=f"rm -rf {rgw_tests}"

also please remove venv dir as well.. sorry for not mentioning it in the requirement..
you can execute this command in this for-loop or below for-loop.. 
cmd=f"rm -rf /home/cephuser/venv"

Reply:
Hi @hmaheswa ,

I have updated PR with requested updates.
This is how updated changes will work.

- find {cleanup_dir}: Starts the search in the {cleanup_dir} directory.
- type d: Restricts the search to directories only (not files).
- \( -name rgw-tests -o -name rgw-ms-tests -o venv \): Looks for directories named rgw-tests or rgw-ms-tests or venv (the -o operator is logical OR).
- exec rm -r {} +: For each directory found, the rm -r command is executed to delete the directory and all its contents (subdirectories and files). {} is a placeholder for the found directory, and + at the end ensures that rm -r is run with multiple directories at once (for efficiency).

Please let me know if any changes or enhancements are needed.

Reply:
Great!
thanks a lot for the changes @obannak